backend/api/views.py

```
@@ -14,23 +15,30 @@
 class CustomUserViewSet(viewsets.ModelViewSet):
     queryset = CustomUser.objects.all()
     serializer_class = CustomUserSerializer
+    permission_classes = [IsAuthenticated]
     
-    # Removed permission_classes
     def get_serializer_class(self):
         if self.action == 'retrieve':
             return CustomUserDetailSerializer
         return CustomUserSerializer
+    
+    def get_queryset(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_queryset`

(ANN201)